src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -197,7 +218,15 @@ int main(int argc, char *argv[])
                                        << std:: endl;
         std::cout.width(26); std::cout << std::internal << "XCLMGMT: " << sensor_tree::get<std::string>( "runtime.build.xclmgmt", "N/A" ) 
                                        << std::endl;
-        return 1;
+        
```

Comment:
I think requiring a value for INTERNAL_BUILD to be true is confusing.  Either INTERNAL_BUILD is defined or it is not.  If latter, then following previous comment I'd probably implement
```
inline bool getenv_or_null(const char* env)
{ return value_or_default(getenv(env),nullptr); }

```
then call as
`if (!getenv_or_null("INTERNAL_BUILD")) { ... }`

Saves string comparison.

Reply:
Thanks for the input. I have made the changes, but I am waiting for Sarabjeet's xclLog API changes PR to get merged so that I can remove xclLog from shim class.